Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -26,6 +26,7 @@
   --remotion-themed-border-radius: 2px;
   --remotion-themed-border-width: 1px;
   --remotion-themed-border: var(--remotion-themed-border-width) solid var(--color-standard-border);
+  --remotion-themed-grid-border: var(--remotion-themed-border-width) solid var(--color-standard-separator);
```

Comment:
Please move this into the formgrid.css and make a formgrid-specific name. See BocList-branch. It's a setting that's just relevant to FormGrid.